noted/recommendations/v1/recommendations.proto

```diff
@@ -24,3 +25,11 @@ message ExtractKeywordsBatchRequest {
 message ExtractKeywordsBatchResponse {
     repeated ExtractKeywordsResponse keywords_array = 1;
 }
+
+message SummarizeRequest {
+    string content = 1;
+}
+
+message SummarizeResponse {
+    string result = 1;
```

Comment:
I'm having trouble understanding what will be stored within `result`. I feel like a more precise variable name would help here. Is `result` a bunch of keywords? a structured string? something else?

Reply:
We can call it "summary" if that's better

Reply:
```suggestion
    // Small text output which summarizes the original entry.
    string summary = 1;
```
WDYT?